index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
-    var i = { size: 4, blocks: [0x0F00, 0x2222, 0x00F0, 0x4444], color: 'purple'   };
-    var j = { size: 3, blocks: [0x44C0, 0x8E00, 0x6440, 0x0E20], color: 'pink'   };
-    var l = { size: 3, blocks: [0x4460, 0x0E80, 0xC440, 0x2E00], color: 'salmon' };
```

Comment:
On the pull request, click Files changed
    Hover over the line of code where you'd like to add a comment, and click the blue comment icon
    In the comment window, type your comment
    Click Start a review
        If you have already started a review, you can click Add review comment
    Above the changed code, click Review changes
    Type a comment summarizing your feedback on the proposed changes
        Note: The problem here is that too many Tetris pieces are the same color. In your review, you might want to suggest that I choose colors which would be easier to see when playing the game.
    Select Request changes to submit feedback introducing changes necessary before merging
    Click Submit review